### PR TITLE
Feat: Added Landing page and Documentation Setup

### LINKED ISSUE #5: Landing Page and Documentation Page Setup
# Landing Page and documentation page setup

## Checklist 

- [x] Deploy on Netlify
- [x] Create Project Structure
- [x] Doc page setup
- [x] Create Landing page

### PR DESCRIPTION
**Pull request by pnchinmay**
#### ISSUE TYPE

- Improvement/Feature implementation

#### CHECKLIST

- [x] Added Landing Page
- [x] Added base style sheets for documentation and components
- [x] Added Installation Page
- [x] Added meta tags for Aesthetic UI

#### LINKED ISSUES

closes #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,95 +1,129 @@
...
+  <meta property="og:image"
+    content="https://raw.githubusercontent.com/pnchinmay/aesthetic-ui/development/assets/logo.png">
+
+  <!-- Twitter -->
+  <meta property="twitter:card" content="summary_large_image">
+  <meta property="twitter:url" content="https://aesthetic-ui-dev.netlify.app/">
+  <meta property="twitter:title" content="Aesthetic UI">
+  <meta property="twitter:description"
+    content="Aesthetic UI, a UI component library for shipping code faster.
+Aesthetic UI is a free, open source framework that provides ready-to-use frontend components that you can easily combine to build responsive web interfaces.">
```
Comment (haarsh24) [inline index.html:38]: Add formatting

**Review by haarsh24 — APPROVED** [review]
Add formatting , Good to Merge.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: index.html (hunk 1 above)
Comment (pnchinmay) [reply index.html:38]: Done

**Pull request merged**